skimage/exposure/exposure.py

```diff
@@ -463,3 +463,28 @@ def adjust_sigmoid(image, cutoff=0.5, gain=10, inv=False):
 
     out = (1 / (1 + np.exp(gain * (cutoff - image / scale)))) * scale
     return dtype(out)
+
+
+def is_low_contrast(image, threshold_percent=0.5, lower_bound=1,
+                    upper_bound=99, method='linear'):
+    """Detemine if an image is low contrast.
+
+    Parameters
+    ----------
+    image : array-like
+        The image under test.
+    threshold_percent : float, optional
+        The low contrast threshold in percent.
```

Comment:
Still not happy with this parameter name. Also, it's not a percent but a fraction. Also, 0.5 is really *not* low contrast, 0.05 is. =)